### DISCUSSION TITLE
Timeline: When timeline minimized, left Bottom corner tool buttons obscures

### DISCUSSION THREAD

**Discussion by franckaubry** [cables-gl/cables_docs#246] [! Welcome - readme first!]
As you can se from the screenshots, when minimized, the toolbar obscures the timeline "Timing" button.

Maximized:
![image](https://user-images.githubusercontent.com/97125854/169978238-ab8aace8-d26a-4af7-ade8-07f276781ebe.png)

Minimized:
![image](https://user-images.githubusercontent.com/97125854/169978203-a61411fe-0158-4b21-88ab-7be84856d684.png)

**Comment by steam0r** [comment]
hey, thanks for the input, we fixed this on dev.cables.gl and it will be in the next release
